app/main.py

```diff
@@ -1,3 +1,14 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable, Any
+
+
+def cache(func: Callable) -> Any:
```

Comment:
are u sure about type annotation for return type? it seems to me, that cache func returns callable

Reply:
u r right